### ISSUE TITLE
Expected behavior of iBOT loss?

### ISSUE THREAD

**Issue by deeplearningmaniac** [facebookresearch/dinov2#275]
Hi, I've been running some experiments and had a question about the expected behavior of the losses, in particular the iBOT loss. As can be seen in the attached image, the iBOT loss seems to tempoarily go up and then eventually goes back down. I was wondering if this is expected behavior? So far I've tested at two different learning rates and the behavior occurs with both. Thank you!

![Screenshot 2023-10-19 at 12 17 04 PM](https://github.com/facebookresearch/dinov2/assets/148477540/328e8b84-89ca-4b4f-8fff-a301b0663f7e)

**Comment by lukx1994** [comment]
+1，I meet the same problem, have you solved it?
![W B Chart 2023_11_2 15_35_38](https://github.com/facebookresearch/dinov2/assets/49824348/ece016e3-9a6d-431c-abd1-a1ccd77f7fe1)

**Comment by qasfb** [comment]
This is the expected behavior.
I want to point out that this method is self-supervised, so the loss does not only decrease during training.
